scripts/install-build-deps

```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
 if ! $use_ansible ; then
   case $(distro_type) in
       redhat)
-        yum-config-manager --enable powertools
+        source /etc/os-release
+        if [[ $NAME == "Rocky Linux" ]];
+        then
+	    if [[ ! `yum repolist | grep Power` ]]
+            then
+                yum-config-manager --enable powertools
+                yum -y install epel-release
+            fi
+	elif [[ $NAME == "Oracle Linux Server" ]]
```

Comment:
That's cool, did you check it? Where?

Reply:
yes, epel have to installed separately.

Reply:
config.vm.box = "generic/oracle8"
  Oracle linux vagrant box is available for public use.